### PR TITLE
Create 2015-04-18-fsharp-core-notes.md

### PR DESCRIPTION
**Pull request by dsyme**

### CODE REVIEW

File: _posts/2015-04-18-fsharp-core-notes.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+* * PCL libraries*. For F# portable PCL library components, it is normal to use the following text in the project file:
+
+       <Reference Include="FSharp.Core">
+         <Name>FSharp.Core</Name>
+         <AssemblyName>FSharp.Core.dll</AssemblyName>
+         <HintPath>$(MSBuildExtensionsPath32)\..\Reference Assemblies\Microsoft\FSharp\.NETCore\$(TargetFSharpCoreVersion)\FSharp.Core.dll</HintPath>
+       </Reference>
+
+* *Legacy PCL libraries*. "Legacy" portable projects (profile 49) use `.NETPortable` instead of `.NETCore`:
```
Comment (ovatsus) [inline _posts/2015-04-18-fsharp-core-notes.md]: I think you mean 47

File: _posts/2015-04-18-fsharp-core-notes.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+- VS2012 installs FSharp.Core 4.3.0.0 only
+- VS2013 installs FSharp.Core 4.3.1.0 only
+- VS2015 installs FSharp.Core 4.4.0.0 only
+
+This means that if you can assume a particular version of Visual Studio (e.g. in a dev/test situation), then you may
+be able to assume FSharp.Core is in the GAC.  But it is best to avoid this assumption and instead make sure FSharp.Core
+is deployed as part of your application.  To do this, use ``<Private>true</Private>`` in your project file 
+(in the Visual Studio IDE this is equivalent to setting the `CopyLocal` property to `true`  properties for the `FSharp.Core` reference)
+
+## FSharp.Core library verison numbers
```
Comment (forki) [inline _posts/2015-04-18-fsharp-core-notes.md]: small typo: version

File: _posts/2015-04-18-fsharp-core-notes.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+## Do not assume any specific version of FSharp.Core is in the GAC, even if it is on your machine
+
+Once again, do not rely on FSharp.Core being in the GAC.  For F# executable components (see above), 
+use `Private=True` for your executable components.
+
+On some installations of F#, some versions of FSharp.Core are added to the GAC.  Do not rely on these
+being present in production code being deployed off your machine.
+
+There _are_ some execptions to this.
```
Comment (forki) [inline _posts/2015-04-18-fsharp-core-notes.md]: execptions

File: _posts/2015-04-18-fsharp-core-notes.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+## FSharp.Core in Xamarin apps
+
+FSharp.Core is referenced as a CopyLocal component in Xamarin apps for mobile devices.  This reference is done via 
+the nuget package for FSharp.Core, see below.
+
+## The FSharp.Core nuget package
+
+FSharp.Core is also available [as a NuGet package](http://www.nuget.org/packages/FSharp.Core). 
+It is not yet normal to reference FSharp.Core via this package, though it is
+used by Xamarin application templates and by some F# library packages (as a transtive nuget dependency).
```
Comment (forki) [inline _posts/2015-04-18-fsharp-core-notes.md]: transitive

**Comment by forki** [comment]
great to see such a guideline. Could you please add a small bit about your opinions / findings with ILMerge(ing) FSharp.Core? IIRC the compiler can also do a bit like this with the --standalone. I think this could also help.

**Comment by forki** [comment]
oh I didn't see this was already merged ;-)

**Comment by dsyme** [comment]
ok, thanks, adding that now

**Pull request merged**